egs2/wsj0_2mix/enh1/conf/tuning/train_enh_asteroid_DPTNet.yaml

```diff
@@ -0,0 +1,57 @@
+optim: adam
+init: none
+max_epoch: 100
+batch_type: folded
+batch_size: 2 # batch_size 16 can be trained on 4 RTX 2080ti
+iterator_type: chunk
+chunk_length: 32000
+num_workers: 4
+optim_conf:
+    lr: 1.0e-03
+    eps: 1.0e-08
+    weight_decay: 0
+patience: 4
+val_scheduler_criterion:
+- valid
+- loss
+best_model_criterion:
+-   - valid
+    - si_snr
+    - max
+-   - valid
+    - loss
+    - min
+keep_nbest_models: 1
+scheduler: reducelronplateau
+scheduler_conf:
+    mode: min
+    factor: 0.5
+    patience: 1
+model_conf:
+    loss_type: si_snr
+encoder: same
+decoder: same
+separator: asteroid
+separator_conf:
+    num_spk: 2
+    model_name: DPTNet
+    # Following the model related args
+    n_src: 2
```

Comment:
Having both `num_spk` and `n_src` doesn't seem optimal.

Reply:
Thanks Manuel. This is because our num_spk is the default arg in AbsSeparator. However, I saw "n_src" is not in every Asteroid model, e.g. some enhancement only models. So I can't find a unified way to pass the "num_spk" to the "n_src". And I just left the whole model-related args which may contain n_src.   Any idea about this part? Maybe I could check getattr(asteroid_model, "n_spk")?

Reply:
> However, I saw "n_src" is not in every Asteroid model

True. 

As you'd like, really :wink: